tests/unit-tests/13-modules/#.scm

```diff
@@ -17,13 +17,37 @@
                              calls))))
            namespaces)))
 
+(define ##output-port (open-string))
+
+(define (call-with-output-string-port port thunk)
```

Comment:
Should be called `call-with-output-port` because `port` could be any kind of output port.

Reply:
This function only works on `string-port` or `vector-port`.